bin/generate_contributors.go

```diff
@@ -36,87 +36,78 @@ func main() {
 	dateInput, _ := bufio.NewReader(os.Stdin).ReadString('\n')
 	lastReleaseDate := fmt.Sprintf("%s%s", dateInput[:len(dateInput)-1], "T12:00:00.000Z")
 
-	// (HACK): Generate list of Sourcegraphers to exclude from the list of contributors
-	//
-	// To manually re-generate the `sourcegraphers` map:
-	// Go to https://github.com/orgs/sourcegraph/teams/everyone/members
-	// Run this code in the console
-	//
-	// 		let sourcegraphers = [];
-	// 		window.document.querySelectorAll('li.table-list-item').forEach(
-	//   		el => sourcegraphers.push(`"${el.dataset.bulkActionsId.trim()}": "",`)
-	// 		)
-	// 		copy(`sourcegraphers := map[string]string{\n${sourcegraphers.join('\n  ')}\n}`)
-	//
-	// TODO: Replace with API call as part of the running of this script
-	sourcegraphers := map[string]string{
-		"adaaaam":         "",
-		"aileenrose":      "",
-		"attfarhan":       "",
-		"beyang":          "",
-		"bot":             "",
-		"christinaforney": "",
-		"creachadair":     "",
-		"dadlerj":         "",
-		"ebrodymoore":     "",
-		"efritz":          "",
-		"eseliger":        "",
-		"felixfbecker":    "",
-		"gbrik":           "",
-		"ggilmore":        "",
-		"ijt":             "",
-		"KattMingMing":    "",
-		"keegancsmith":    "",
-		"kevinchen94":     "",
-		"kevinzliu":       "",
-		"kzh":             "",
-		"lguychard":       "",
-		"mrnugget":        "",
-		"nicksnyder":      "",
-		"renovate":        "",
-		"renovate[bot]":   "",
-		"rrhyne":          "",
-		"ryan-blunden":    "",
-		"ryanslade":       "",
-		"rvantonder":      "",
-		"slimsag":         "",
-		"sqs":             "",
-		"todo":            "",
-		"tsenart":         "",
-		"unknwon":         "",
-		"uwedeportivo":    "",
-		"vanesa":          "",
-	}
-
 	issueDate, err := time.Parse(time.RFC3339, lastReleaseDate)
 	if err != nil {
 		log.Fatal("error parsing date", err)
 		return
 	}
-	contributors := make(map[string]IssueAuthorDetails)
+
+	// Get members of our team
+	teamOpt := &github.TeamListTeamMembersOptions{Role: "all", ListOptions: github.ListOptions{PerPage: 100, Page: 1}}
+	sourcegraphTeamMembers, _, err := client.Teams.ListTeamMembersBySlug(ctx, "sourcegraph", "everyone", teamOpt)
+
+	// Extract teammate usernames
+	sourcegraphUsernames := make(map[string]struct{}, len(sourcegraphTeamMembers))
+
+	for _, m := range sourcegraphTeamMembers {
+		sourcegraphUsernames[*m.Login] = struct{}{}
+	}
+
+	contributors := make(map[string]ContributorDetails)
 	page := 0
 
 	// First print via repo
 	for _, repo := range repos {
+		page = 0
```

Comment:
This value is already zero due to assignment above

Reply:
```suggestion
```